R/ab_scenario.R

```diff
@@ -223,7 +220,7 @@ ab_json = function(
 
 #' Save OD data as JSON files for import into A/B Street
 #'
```

Comment:
:+1: the info was in there but just needed the right structure and Roxygen2 tags. Efficient fixes!

Reply:
should we use this then for the `@return`` tag?